docs/custom_loading.rst

```diff
@@ -77,7 +77,8 @@ ensure that the data file being loaded is compatible with the loader function.
                 os.path.splitext(args[0].lower())[1] == '.fits')
 
 
-    @data_loader("my-format", identifier=identify_generic_fits)
+    @data_loader("my-format", identifier=identify_generic_fits,
```

Comment:
This is still missing a discussion of the "generic" vs "custom" distinction we were discussing.

Reply:
Deferred to a follow-on PR